modules/gltf/gltf_document.cpp

```diff
@@ -3008,16 +3008,18 @@ Error GLTFDocument::_parse_images(Ref<GLTFState> state, const String &p_base_pat
 		} else if (mimetype == "image/jpeg") { // Loader buffer as JPEG.
 			ERR_FAIL_COND_V(Image::_jpg_mem_loader_func == nullptr, ERR_UNAVAILABLE);
 			img = Image::_jpg_mem_loader_func(data_ptr, data_size);
-		} else {
-			// We can land here if we got an URI with base64-encoded data with application/* MIME type,
-			// and the optional mimeType property was not defined to tell us how to handle this data (or was invalid).
-			// So let's try PNG first, then JPEG.
+		}
+        // We can land here if we got an URI with base64-encoded data with application/* MIME type,
+        // and the optional mimeType property was not defined to tell us how to handle this data.
+        // This also lets us handle bogus mime types (e.g. `image/jpeg` but the data is actually a PNG).
+        // That's not as per spec but bogus glb files do exist in production so we're lenient.
```

Comment:
Wrong indentation, but if you put this comment in the main scope, you'll have to change the phrasing.

It's no longer a "we can land here" as all code goes through this scope. "here" referred to after `if (img.is_null())`.